.gitignore

```diff
@@ -1,4 +1,5 @@
 /.idea/
 /configs.php
 /vendor/
-/.phpunit.result.cache
\ No newline at end of file
+/.phpunit.result.cache
+/public/node_modules
```

Comment:
Nitpick.
Add a line feed to the end of the file. You can configure your editor to automatically add a line feed on save.